src/components/common/ProductImg/ProductImg.tsx

```diff
@@ -0,0 +1,10 @@
+import { memo, ImgHTMLAttributes } from "react";
+import empty from "../../../assets/img/empty.jpg";
+import Styled from "./ProductImgStyled";
+
+const ProductImg = ({ src, ...props }: ImgHTMLAttributes<HTMLImageElement>) => {
+  if (!src) src = empty;
```

Comment:
상위 props 를 재정의하는 것은 좋지 않습니다!

```suggestion
const ProductImg = ({ src = empty, ...props }: ImgHTMLAttributes<HTMLImageElement>) => {
```

Reply:
그냥 default value를 지정해주면 작동하지 않아서 찾아보니 default value를 주고싶은 prop에 옵셔널을 지정해주라고 하더라구요! 그래서 
```
interface ProductImgProps extends ImgHTMLAttributes<HTMLImageElement> {
  src?: string;
  theme: object;
}

const ProductImg = ({ theme, src = empty, ...props }: ProductImgProps) => {
  return <Styled.Img {...props} theme={theme} src={src} loading="lazy" />;
};
```
이렇게 했습니다. 하지만 여전히 작동하지 않아서 보니까 아무것도 없는 "" 이 값도 값이 있다고 인식되는 것 같더라구요. 그래서 
```
<ProductImg
        theme={imgTheme}
        src={imageUrl ? imageUrl : undefined}
        alt={name}
      />
```
이런식으로 imageUrl이 존재하지 않으면 undefined 값을 보내도록 해보았습니다. 괜찮을까요?

Reply:
아 옵셔널 props 가 아니었군요?! 고걸 못봤네요! 그러면 그냥 src={src || empty} 요렇게 했어도 됐을 것 같습니다!
재정의 제외하고는 어느 방식이나 개발자 취향이니 루루가 원하시는 방향으로 진행하시면 될 것 같아요! ㅎㅎ